src/main/java/com/internet/shop/model/Role.java

```diff
@@ -0,0 +1,31 @@
+package com.internet.shop.model;
+
+public class Role {
+    private Long id;
+    private RoleName roleName;
+
+    public Role(RoleName roleName) {
```

Comment:
let's make this constructor private and make static method of() as in Bohdans video

Reply:
Done